cfme/tests/test_utilization_metrics.py

```diff
@@ -136,8 +136,7 @@ def query_metric_db(appliance, provider, metric, vm_name=None, host_name=None):
 
 
 # Tests to check that specific metrics are being collected
-@pytest.mark.uncollectif(
-    lambda provider: current_version() < "5.7" and provider.type == 'gce')
+@pytest.mark.meta(blockers=[BZ(1525296, unblock=lambda provider: provider.category != 'cloud')])
```

Comment:
REQUIRED: please use provider.one_of(CloudProvider) instead

Reply:
@izapolsk improve as you suggested.